frontend/src/metabase/visualizations/echarts/cartesian/option/axis.ts

```diff
@@ -7,149 +7,21 @@ import type {
   RenderingContext,
 } from "metabase/visualizations/types";
 import type {
+  AxisFormatter,
   CartesianChartModel,
   Extent,
+  YAxisModel,
 } from "metabase/visualizations/echarts/cartesian/model/types";
 
-import type {
-  AxesFormatters,
-  AxisFormatter,
-  AxisRange,
-} from "metabase/visualizations/echarts/cartesian/option/types";
 import { CHART_STYLE } from "metabase/visualizations/echarts/cartesian/option/style";
 
-import { getMetricDisplayValueGetter } from "metabase/visualizations/echarts/cartesian/model/dataset";
+import type { ChartMeasurements } from "metabase/visualizations/echarts/cartesian/option/types";
 import { isNumeric } from "metabase-lib/types/utils/isa";
 
 const NORMALIZED_RANGE = { min: 0, max: 1 };
 
-const getYAxisTicksWidth = (
```

Comment:
Moved to https://github.com/metabase/metabase/pull/36879/files#diff-2225bf3348720497bacb168a0c058e6ab7058d598e51d175312a8dd7a06d69ff